core/templates/css/oppia.css

```diff
@@ -434,7 +434,7 @@ a:focus {
 }
 
 /* Change all the borders of an input to red when there is validation errors. */
-input.ng-dirty.ng-invalid, md-input-group.md-default-theme input.ng-dirty.ng-invalid {
+input.ng-dirty.ng-invalid, textarea.ng-dirty.ng-invalid, md-input-group.md-default-theme {
```

Comment:
Previously, we didn't have any red border around `textarea` objects when the value was invalid.